syft/messaging/plan.py

```diff
@@ -293,19 +418,17 @@ def _create_placeholders(self, args_shape):
     def _known_workers(self):
         return self.owner._known_workers
 
-    # Override property of nn.Module
     @property
-    def owner(self):
-        return self._owner
-
-    @owner.setter
-    def owner(self, new_owner):
-        self._owner = new_owner
+    def location(self):
+        raise AttributeError("Plan has no attribute location")
 
-    # Override property of nn.Module
+    # For backward compatibility
     @property
-    def location(nn_self):
-        raise AttributeError("Plan has no attribute location")
+    def readable_plan(self):
+        return self.procedure.commands
+
+    def parameters(self):
```

Comment:
It might be a bit confusing to call this `parameters()` if it only returns the state tensors and not the parameters given when calling the plan

Reply:
will add a comment: I'm trying here to match the torch api of nn.Module where .parameters() return the model parameters